source/API_Reference/Web_API_v3/bounces.apiblueprint

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+weight: 100
+title: Bounces API
+seo:
+  description: Manage your bounce list
+  title: Bounce API
+navigation:
+  show: true
+---
+FORMAT: 1A
+
+# Bounces
+
+
+## Bounces Collection [/v3/suppression/bounces?start_time={start_time}&end_time={end_time}]
+
++ Parameters
+
+    + start_time (optional, number, `1443651141`) ... Refers start of the time range when a bounce was created.
+    + end_time (optional, number, `1443651154`) ... Refers end of the time range when a bounce was created.
+
+### List all bounces [GET]
+
++ Response 200 (application/json)
+
+        [
+            {
+                "created": 1443651125,
+                "email": "testemail1@test.com",
+                "reason": "550 5.1.1 The email account that you tried to reach does not exist. Please try double-checking the recipient's email address for typos or unnecessary spaces. Learn more at  https://support.google.com/mail/answer/6596 o186si2389584ioe.63 - gsmtp ",
+                "status": "5.1.1"
+            },
+            {
+                "created": 1433800303,
+                "email": "testemail2@testing.com",
+                "reason": "550 5.1.1 <testemail2@testing.com>: Recipient address rejected: User unknown in virtual alias table ",
+                "status": "5.1.1"
+            }
+        ]
+
+## Bounces Collection [/v3/suppression/bounces]
+
+There's two bounce delete options:
+1) deleting all bounces by specifying "delete_all" to true in the request body, or
+2) deleteing some emails by specificy "emails" and an array of emails in the response body.
+
+### Delete bounces [DELETE]
+
++ Request (application/json)
+
+        {
```

Comment:
You missing a `+ Body`. This should be in the following format:

`+ Request (application/json)

    + Body

            {
                "delete_all": false
            }
`